evap/staff/tests/test_views.py

```diff
@@ -7,8 +8,8 @@
 from model_mommy import mommy
 import xlrd
 
-from evap.evaluation.models import Semester, UserProfile, Course, CourseType, \
-                                   TextAnswer, Contribution
+from evap.evaluation.models import Semester, UserProfile, Course, CourseType, TextAnswer, Contribution, Questionnaire,\
+    Question
```

Comment:
I disagree as this is the standard import indentation style(PyCharm) which also decreases the number of import lines to a minimum and imo doesn't reduce readability